components/support/nimbus-fml/fixtures/ios/tests/script.swift

```diff
@@ -0,0 +1,11 @@
+import FeatureManifest
```

Comment:
This is … suprising. I'm not sure I understand why this is necessary.

Reply:
This is only for the tests, the reason is that the scripts are run after the FeatureManifest, and all the Nimbus helpers are compiled together into a module that's later imported by the script. In the app, the FeatureManifest would be compiled together with the app so this wouldn't be needed